tests/pipelines/test_pipelines_common.py

```diff
@@ -378,7 +378,7 @@ def test_inference_batch_consistent(self, batch_sizes=[2]):
         self._test_inference_batch_consistent(batch_sizes=batch_sizes)
 
     def _test_inference_batch_consistent(
-        self, batch_sizes=[2], additional_params_copy_to_batched_inputs=["num_inference_steps"]
+        self, batch_sizes=[2], additional_params_copy_to_batched_inputs=["num_inference_steps"], batch_generator=True
     ):
```

Comment:
open muse does not support batched generators

Reply:
Is it concerning?

Reply:
wdym?

Reply:
OpenMUSE not supporting batched generators -- is it concerning?

Reply:
Not really, I think it's fine. I think the batched generators behavior is very diffusers specific and requires custom handling for each place a generator is used. We have a special helper for the initial latent generation on other pipelines that helps with this, but I don't think we should expect every pipeline to handle as such if they have more places where they use rng